frontend/src/pages/add/BookAdd.tsx

```diff
@@ -145,71 +143,68 @@ class BookAdd extends React.Component<{}, BookAddState> {
     }
   };
 
-  onSubmit = (): void => {
-    alert("A form was submitted: \n" + this.state.inputBoxText);
+  const onSubmit = (): void => {
+    console.log(textBox);
```

Comment:
I don't know if it is good practice to console.log(), but for testing its fine. Can you maybe note it down to remove later in case we forget.

Reply:
I left it as a placeholder from the alert, just to simplify our code. When we go back to change the onSubmit() function we can remove it. It is actually good practice to log, but I set up a bunch of logging on another branch that I'm now working on